include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -52,8 +54,7 @@ class BTFactory {
     static std::string currentTree;
     static std::string keeperTree;
     static bool weMadeTrees;
-
-    static std::shared_ptr<bt::BehaviorTree> testing_tree;
+    static std::shared_ptr<rtt::ai::analysis::Play> play;
```

Comment:
Personally am not a huge fan of using static variables unless necessary, which I dont' think it is here. BtFactory is already a static class.

Reply:
The variable belongs to the class. Therefore it needs to be static, right?